mathics/format/json.py

```diff
@@ -59,10 +59,7 @@ def graphics_3D_elements(self, **options) -> list:
     result = []
     for element in self.elements:
         format_fn = lookup_method(element, "json")
-        if format_fn is None:
-            result += element.to_json()
-        else:
-            result += format_fn(element)
+        result += format_fn(element)
```

Comment:
this assumes all of the to_json routines have been converted.

Reply:
They do, I made a Ctrl+Shift+F (search in all files) and nothing of `to_json`.